src/components/custom-aggrid/custom-aggrid-header-utils.ts

```diff
@@ -20,7 +20,11 @@ export const makeAgGridCustomHeaderColumn = ({
     const { filterDataType, filterEnums = {} } = filterParams || {};
 
     const customFilterOptions =
-        filterDataType === FILTER_DATA_TYPES.TEXT ? filterEnums[field] : [];
+        filterDataType === FILTER_DATA_TYPES.TEXT
+            ? filterEnums[field]
+            : filterDataType === FILTER_DATA_TYPES.BOOLEAN
+            ? ['true', 'false']
```

Comment:
Then we don't list the actual values in the data, is it ok from a functional point of view ?

Reply:
i think we should not handle a boolean filter as an enum, for a boolean column its always yes or no